Edited change from this commit:

Update of /cvsroot/cpufreqd/sources2/src
In directory sc8-pr-cvs4.sourceforge.net:/tmp/cvs-serv5102/src

Modified Files:
	cpufreqd_acpi.c cpufreqd_acpi_ac.c cpufreqd_acpi_ac.h 
	cpufreqd_acpi_battery.c cpufreqd_acpi_battery.h 
	cpufreqd_acpi_event.c cpufreqd_acpi_event.h 
	cpufreqd_acpi_temperature.c cpufreqd_acpi_temperature.h 
	cpufreqd_cpu.c cpufreqd_programs.c 
Log Message:
- make the acpi thermal comaptible with kernels >2.6.25 when the type has
  changed from "ACPI Thermal Zone" to "acpitz"
- squash some converison warnings (gcc 4.3 is very loud about it)


Index: cpufreqd-2.3.3/src/cpufreqd_acpi_temperature.c
===================================================================
--- cpufreqd-2.3.3.orig/src/cpufreqd_acpi_temperature.c	2008-09-08 23:57:02.961182316 +0900
+++ cpufreqd-2.3.3/src/cpufreqd_acpi_temperature.c	2008-09-09 00:01:56.361181870 +0900
@@ -26,9 +26,11 @@
 #include "cpufreqd_acpi.h"
 #include "cpufreqd_acpi_temperature.h"
 
-#define THERMAL "thermal"
-#define THERMAL_TYPE "ACPI thermal zone"
-#define THERMAL_TEMP "temp"
+#define THERMAL			"thermal"
+#define THERMAL_TYPE		"acpitz"
+/* the below is for kernels <= 2.6.25 */
+#define THERMAL_TYPE_ALT	"ACPI thermal zone"
+#define THERMAL_TEMP		"temp"
 
 struct thermal_zone {
 	int temperature;
@@ -80,6 +82,9 @@ static int atz_callback(struct sysfs_cla
 int acpi_temperature_init(void)
 {
 	find_class_device(THERMAL, THERMAL_TYPE, atz_callback);
+	/* try with the old type name */
+	if (atz_dir_num <= 0)
+		find_class_device(THERMAL, THERMAL_TYPE_ALT, atz_callback);
 	if (atz_dir_num <= 0) {
 		clog(LOG_INFO, "No thermal zones found\n");
 		return -1;
